src/Schema/Events/System.ts

```diff
@@ -0,0 +1,29 @@
+import { ActionType } from "../Event"
+import { PageOwnerType } from "../OwnerType"
+
+/**
+ * Schemas describing system events
+ * @packageDocumentation
+ */
+
+/**
+ * A user has been on a page for 15 seconds.
+ *
+ * This schema describes events sent to Segment from [[timeOnPage]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "timeOnPage",
+ *    category: "15 seconds",
+ *    context_page_owner_type: "home"
+ *  }
+ * ```
+ */
+export interface TimeOnPage {
+  action: ActionType.timeOnPage
+  category: "15 seconds"
```

Comment:
Is 15 seconds the only option available?

Reply:
Yes it's the only event we want to fire for right now! We might decide to add another time option later, but for now, we are trying to cut back on events since we keep maxing out our Segment API limit.